.gitignore

```diff
@@ -103,3 +103,6 @@ safepaths.realm
 
 # swift packages
 ios/COVIDSafePaths.xcworkspace/xcshareddata/swiftpm/Package.resolved
+
+# Firebase
+android/app/google-services.json
```

Comment:
The app doesn't compile without a `google-services.json` file, right?
I can add it to the env configuration, just wanted to confirm if it's required

Reply:
Yeah this sounds about right, is it possible to do something like the examples GoogleServices on the `ios` folder?

Reply:
Sure!

Reply:
Yes, it is requred.  I will rebase.

Reply:
I got these error messages:
```
src/More/About.spec.tsx:8:8 - error TS2307: Cannot find module 'react-native-device-info' or its corresponding type declarations.

8 } from "react-native-device-info"
         ~~~~~~~~~~~~~~~~~~~~~~~~~~

src/More/About.tsx:15:8 - error TS2307: Cannot find module 'react-native-device-info' or its corresponding type declarations.

15 } from "react-native-device-info"
          ~~~~~~~~~~~~~~~~~~~~~~~~~~

```

Reply:
Try running `yarn install`, I had the same error some minutes ago

Reply:
We will need to commit a fake `google-services.json` to compile the app
This line won't be needed anymore